Makefile

```diff
@@ -181,7 +181,7 @@ docs:
 
 website-image: docs
 
-$(SERVICES_IMAGES): %-image: $(SERVICES_IMAGE_DEPS) $(shell git ls-files $$*)
+$(SERVICES_IMAGES): %-image: $(SERVICES_IMAGE_DEPS) $(shell git ls-files $$* -- . ':!:deployment.yaml')
```

Comment:
I can't get this to work on my laptop, are you sure this works? I'd expect you to need to use grep or something to remove things.

Reply:
I was missing some `**/` but yes you can use git's [pathspec](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefpathspecapathspec) here to exclude (`!`) some patterns.

Reply:
This works for me now, `make deploy` auth then `touch auth/deployment.yaml` doesn't rebuild the image